Libraries/Components/RefreshControl/RefreshControl.js

```diff
@@ -137,11 +153,11 @@ const RefreshControl = React.createClass({
   },
 
   _onRefresh() {
+    this._lastNativeRefreshing = true;
+
     this.props.onRefresh && this.props.onRefresh();
 
-    if (this._nativeRef) {
-      this._nativeRef.setNativeProps({refreshing: this.props.refreshing});
-    }
+    this.forceUpdate();
```

Comment:
If the user calls `setState`, we'll run a render pass and `componentDidUpdate` will be called by React as usual, so this `forceUpdate` call is necessary only when the user doesn't re-render the RefreshControl (I think?).

Can you add a short comment explaining that we want to update this component to reconcile the refreshing state between native and JS whether or not the user updates this component in the `onRefresh` callback?